docs/design-system/tokens/fonts.css

```
@@ -0,0 +1,7 @@
+/* ArtisanPack UI — Webfonts
+   Heading + Body: Poppins
+   Code / mono: Space Mono
+   Both from Google Fonts. To self-host, swap this @import for local
+   @font-face rules pointing at assets/fonts/. */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap'" (import-notation)

(import-notation)